Webhooks/Edify.ashx

```diff
@@ -159,7 +162,29 @@ namespace RockWeb.Plugins.rocks_kfs.Webhooks
 
             if ( request.ContentType.Contains( "application/json" ) )
             {
-                ProcessJsonContent( request, response );
+                var edifyWebhookStart = RockCache.Get( "EdifyWebhookStart" );
+                var edifyWebhookEnd = RockCache.Get( "EdifyWebhookEnd" );
+                if ( edifyWebhookStart != null && edifyWebhookEnd != null )
+                {
+                    _previousMilliseconds = ( ( DateTime ) edifyWebhookEnd - ( DateTime ) edifyWebhookStart ).TotalMilliseconds;
+                }
+                if ( _previousMilliseconds >= 500 )
+                {
+                    var updateEndDate = ( DateTime ) edifyWebhookEnd;
+                    var timeToReset = updateEndDate.AddMilliseconds( _previousMilliseconds * 2 );
```

Comment:
Make milliseconds factor configurable const at the top to not hide down in code. And add comment to make it configurable in the future with provider.